bigbluebutton-html5/imports/ui/components/user-list/user-list-content/component.jsx

```diff
@@ -26,7 +27,7 @@ class UserContent extends PureComponent {
       compact,
     } = this.props;
 
-    const showWaitingRoom = (isGuestLobbyMessageEnabled && isWaitingRoomEnabled)
+    const showWaitingRoom = (ALWAYS_SHOW_WAITING_ROOM && isWaitingRoomEnabled)
```

Comment:
Are we getting rid of the `enableGuestLobbyMessage` setting?

if not, the correct condition would be 

```
const showWaitingRoom = ((ALWAYS_SHOW_WAITING_ROOM || isGuestLobbyMessageEnabled) && isWaitingRoomEnabled)
       || pendingUsers.length > 0;
```

Reply:
`enableGuestLobbyMessage` will still be used, but not as a condition to show or hide the waiting users in the sidebar